packages/wrangler/src/config-path.ts

```diff
@@ -0,0 +1,25 @@
+import fs from "node:fs";
+import os from "node:os";
+import path from "node:path";
+import xdgAppPaths from "xdg-app-paths";
+
+export function getConfigPath() {
+	//TODO: We should implement a custom path --global-config
```

Comment:
Or the `WRANGLER_HOME` type environment variable?

Reply:
Mmmm yeah that would be a good one too, I think Wrangler1 had something like that.